client/utils/utils.go

```diff
@@ -62,6 +62,22 @@ func CompleteAndBroadcastTxCLI(txBldr authtxb.TxBuilder, cliCtx context.CLIConte
 		return nil
 	}
 
+	if !cliCtx.SkipConfirm {
+		stdSignMsg, err := txBldr.BuildSignMsg(msgs)
+		if err != nil {
+			return err
+		}
+
+		fmt.Fprintf(os.Stderr, "%s\n\n", cliCtx.Codec.MustMarshalJSON(stdSignMsg))
+
+		buf := client.BufferStdin()
+		ok, err := client.GetConfirmation("confirm transaction before signing and broadcasting", buf)
```

Comment:
I think the confirmation should focus on getting the user to double check the important information, not the entire transaction. These would be:

- For all messages, the fee amount.
- For send message, the recipient. 

Maybe the transaction can be displayed as well, but it's especially important that users check the two above.

Reply:
Ehhh this is pretty complicated @gamarin2. I don't understand why we can just display the (unsigned) tx. It already includes sender/recipient and it already includes the fee and gas -- everything the client should know.

Otherwise, we'll need to have a type switch here on every single msg in the tx and the UX will be ugly.

Reply:
Hmm the point of the original issue was to avoid excessive fees. For a user, the two most important things to check are fees and recipient if msgType = send.

I was hoping a simple display of "This will cost you X atoms in fees", with the conversion uatoms atoms performed for the end user. If you see `"fee":{"amount":"3000000000000000", "denom":"uatom"}}` it's actually hard to check if you aren't paying ten times more fees than expected, which was the original concern.

Reply:
imho, I think simply displaying the unsigned tx is the easiest and best way to go here. If you want, in addition to the unsigned tx, we may _also_ display the fees in atoms.

Reply:
I'm not arguing with easiest, i'm arguing with best :p 

All I'm saying is the point of the original issue was to make sure users do not make a mistake/typo that will end up in them losing funds. I think displaying the fees in atoms is absolutely necessary to achieve that.

Reply:
Sure, but I also would like to add showing the full unsigned tx is _also_ the best. Shall we compromise and additionally add a section to the prompt stating the fees in atoms?

Reply:
Sounds good :+1:

Reply:
Thing is, this now ties into #3510

Reply:
how so?